l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -168,7 +173,9 @@ def _key(tax_id):
                         "AliquotaIVA": aliquota,
```

Comment:
Il price_total è in valuta estera, mi risulta quindi da convertire all'euro, alla riga 144:
```
"Imposta": fpaToEur(
    tax_id.price_total, invoice, euro
),
```